addon/modifiers/did-update.js

```diff
@@ -58,14 +59,30 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
+    capabilities: capabilities(gte('3.22.0') ? '3.22' : '3.13', {
+      disableAutoTracking: gte('3.22.0') ? false : true,
+    }),
 
     createModifier() {
       return { element: null };
     },
-    installModifier(state, element) {
+
+    installModifier(instance, element, args) {
       // save element into state bucket
-      state.element = element;
+      instance.element = element;
```

Comment:
Please revert this. The name of the first variable should not be `instance` (`bucket` is what we commonly use, but `state` is still better than `bucket`).

Reply:
done!

Reply:
does this mean that this should change to state? https://github.com/ember-modifier/ember-modifier/blob/master/addon/-private/class/modifier-manager.ts#L32